--- loggerhead-1.18.1/info.py.old	2011-08-13 10:59:36.731438116 +0100
+++ loggerhead-1.18.1/info.py	2011-08-13 10:59:47.416438124 +0100
@@ -6,7 +6,7 @@
 bzr_plugin_version = (1, 18, 0)
 
 bzr_compatible_versions = [
-    (1, 17, 0), (1, 18, 0), (2, 0, 0), (2, 1, 0), (2, 2, 0), (2, 3, 0)]
+    (1, 17, 0), (1, 18, 0), (2, 0, 0), (2, 1, 0), (2, 2, 0), (2, 3, 0), (2, 4, 0)]
 
 bzr_minimum_version = bzr_compatible_versions[0]
 
